### PR TITLE
Update person form fields

### LINKED ISSUE #1354: Update copy on patient registration form
There are some [copy updates to the patient registration form](https://www.figma.com/file/CEzBM8kUw8LyWdfAhUThYb/Engineering-Ready-Prototype?node-id=2210%3A596) that we would love to implement before patient self-registration launches. 

Specific changes:

- [x] Update instructions about required fields at top of form

- [x] Update explainer text under Demographics

- [x] Update race response options to include slashes instead of "or"

- [x] Rephrase ethnicity question

- [x] Rephrase "Other" fields as questions (I think this is already covered in a different ticket)

### PR DESCRIPTION
**Pull request by maya**
## Related Issue or Background Info

- Fixes #1354

## Add'l info 
- Uses the current USWDS required * markup

### CODE REVIEW

File: frontend/src/app/commonComponents/Required.tsx
Comment (maya) [inline frontend/src/app/commonComponents/Required.tsx:5]: How do I add a space only when there's a label? e.g. "` *`" so `My label *`. And no space when there's no label, e.g. `(*)`

File: frontend/src/app/commonComponents/Required.tsx
Comment (ClydeDroid) [inline frontend/src/app/commonComponents/Required.tsx:7]:
```suggestion
      {props.label ? " *" : "(*)"}
```

File: frontend/src/app/constants/constants.d.ts
Comment (maya) [inline frontend/src/app/constants/constants.d.ts:9]: Is there something else we should use instead of "refused" for "prefers not to answer"?

File: frontend/src/app/constants/index.tsx
Comment (maya) [inline frontend/src/app/constants/index.tsx:62]: Same as https://github.com/CDCgov/prime-simplereport/pull/1356/files#r614427805

File: frontend/src/app/commonComponents/Required.tsx
Comment (maya) [reply frontend/src/app/commonComponents/Required.tsx:5]: Done in feabca7

**Comment by maya** [comment]
@nickscialli-usds this relates to #1303, feel free to edit this PR as needed.

**Comment by nickscialli-usds** [comment]
I won't have time to look at this until later today, but my first thought is #1303 implements the additional options (e.g., "Prefer not to answer") so I'd recommend removing those changes from this PR. Alternatively, you could rebase off of #1303 and wait for that one to land. 

I think that if this branch got merged as-is, some of the new options would actually cause backend errors because they haven't been added to `Translators.java`

File: frontend/src/app/constants/constants.d.ts
Comment (nickscialli-usds) [reply frontend/src/app/constants/constants.d.ts:9]: The reason I started using `"refused"` for these fields is we had been using `"Refused to answer"`, which mapped to `"refused"` on the backend, for Race options. It's not necessarily the right call, but importantly it's being sent over to Report Stream as "unknown" regardless of what string we use here.

Comment (maya) [reply frontend/src/app/constants/constants.d.ts:9]: Thanks for the context

**Comment by maya** [comment]
@nickscialli-usds I reverted those changes in 1ccbd7c so this PR should be mergeable.

File: frontend/src/app/patients/__snapshots__/EditPatient.test.tsx.snap
Comment (nickscialli-usds) [inline frontend/src/app/patients/__snapshots__/EditPatient.test.tsx.snap:1040]: I think this snapshot needs to be updated since you removed "Prefer not to answer" for now

File: frontend/src/app/patients/__snapshots__/EditPatient.test.tsx.snap
Comment (nickscialli-usds) [inline frontend/src/app/patients/__snapshots__/EditPatient.test.tsx.snap:2300]: Again snapshot update?

File: frontend/src/patientApp/timeOfTest/__snapshots__/PatientFormContainer.test.tsx.snap
Comment (nickscialli-usds) [inline frontend/src/patientApp/timeOfTest/__snapshots__/PatientFormContainer.test.tsx.snap:1060]: Snapshot update?

**Review by nickscialli-usds — COMMENTED** [review]
I think this will be good to go after a snapshot update

File: frontend/src/app/patients/__snapshots__/EditPatient.test.tsx.snap
Comment (maya) [reply frontend/src/app/patients/__snapshots__/EditPatient.test.tsx.snap]: Done in 7e4b0f6.

File: frontend/src/patientApp/timeOfTest/__snapshots__/PatientFormContainer.test.tsx.snap
Comment (maya) [reply frontend/src/patientApp/timeOfTest/__snapshots__/PatientFormContainer.test.tsx.snap]: Done in 7e4b0f6.

File: frontend/src/app/patients/__snapshots__/EditPatient.test.tsx.snap
Comment (maya) [reply frontend/src/app/patients/__snapshots__/EditPatient.test.tsx.snap]: Done in cae3f31.

**Comment by maya** [comment]
Thanks for catching those! Snapshots updated and tests are ✅

**Review by nickscialli-usds — APPROVED** [review]
🚀

**Pull request merged**
